packages/dm-core-plugins/src/meta/MetaPlugin.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  EntityView,
+  IUIPlugin,
+  Loading,
+  TGenericObject,
+  TMeta,
+  useDocument,
+} from '@development-framework/dm-core'
+import { Table } from '@equinor/eds-core-react'
+import { DateTime } from 'luxon'
+
+export const MetaPlugin = (props: IUIPlugin) => {
+  const { document, isLoading, error } = useDocument<TGenericObject>(
+    props.idReference,
+    0
+  )
+
+  if (isLoading) return <Loading />
+
+  if (error) throw new Error(JSON.stringify(error, null, 2))
+
+  if (!document) return
+
+  const meta: TMeta = document._meta_
+
+  if (document.type !== 'dmss://system/Plugins/dm-core-plugins/common/Meta')
+    throw new Error(
+      'The meta plugin only supports entities of type "Meta" at this stage'
+    )
+
+  return (
+    <>
+      <Table className={'w-full'}>
+        <Table.Head>
+          <Table.Row>
+            <Table.Cell>Created by</Table.Cell>
+            <Table.Cell>Created time</Table.Cell>
+            <Table.Cell>Last modified by</Table.Cell>
+            <Table.Cell>Last modified time</Table.Cell>
+          </Table.Row>
+        </Table.Head>
+        <Table.Body>
+          <Table.Row>
+            <Table.Cell>{meta?.createdBy ?? 'nil'}</Table.Cell>
```

Comment:
In TablePlugin we use "-" to show null values, feel like it's a bit less distracting

Reply:
Yeah thats better. Fixed!